src/NServiceBus.Core.Tests/Routing/SingleInstanceRoundRobinDistributionStrategyTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿namespace NServiceBus.Core.Tests.Routing
+{
+    using System.Collections.Generic;
+    using NServiceBus.Routing;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class SingleInstanceRoundRobinDistributionStrategyTests
```

Comment:
I think I'll add another assertion to this test instead of creating a new test :+1: